portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,16 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+ * Adds a random fun fact to the page
+  */
+function displayFunFact(){
+  const facts =
+      ['I was born in Weehawken, New Jersey, and lived on a houseboat on the Hudson river!', 'I love the outdoors. My hobbies include trail running, hiking, skiing, camping, and climbing 14ers!', 'I like to wake up with the sun, and am a morning person. I love the quietness of 6am.', 'I really enjoy reading. Urban Fantasy, like the Dresden Files, are some of my favorite reads!'];
 
   // Pick a random greeting.
```

Comment:
these are no longer greetings. Maybe update the comment?

Reply:
Nice catch, updated.